content/hub-services/index.md

```diff
@@ -0,0 +1,125 @@
+---
+# Page title
+title: Hub services
+# Page type - we want a landing page (such as a homepage)
+type: landing
+aliases:
+  - /hub-services/
+
+
+# Your landing page sections - add as many different content blocks as you like
+sections:
+  - block: markdown
+    id: hub-services-intro
+    content:
+      title: Your interactive computing hub, the way you want it
+      text: |
+        At 2i2c, we have extensive expertise in rapidly deploying custom cloud computing hubs to help research and education communities create and share knowledge.
+
+        This page outlines our standard hub configuration options, which can be usually be deployed with a short turnaround of 1 day or less.
+
+        However, should your community require a more customized approach, please don't hesitate to [contact us](../join/_index.md) to discuss your specific requirements.
+
+  - block: markdown
+    content:
+      title: Sign-in
+      text: |
+        We support the following authentication and authorization options:
+          - GitHub - with support for GitHub Organization and Teams
+          - CILogon - with support for institutional logins, Google Auth, Microsoft, and ORCID
+          - Shared Password - simple authentication with a global shared password, ideal for workshops and webinar
+        <figure class="videofigure">
+          {{< video src="videos/jupyterhub-admin.mp4">}}
+
+          <figcaption>
+              Powered by <a href="https://jupyterhub.readthedocs.io"><img src="/images/logos/project/jupyterhub.svg" /></a>
+          </figcaption>
+        </figure>      
+
+  - block: features
+    content:
+      title: Compute
```

Comment:
Rendering poorly in dark mode

![image](https://github.com/user-attachments/assets/7d5d1122-204b-497a-9e81-5d8cf669cbc1)

Reply:
Nice catch! I tried to hack something that I shouldn't have lol

Reply:
Resolved in https://github.com/2i2c-org/2i2c-org.github.io/pull/295/commits/ff4e4977358f5dfd20eb8eced8966e3573682ff8